tests/integration/test_fast_color_refine.py

```diff
@@ -4,17 +4,17 @@
 import os
 import unittest
 
-from color_refinement import fast_color_refine
-from color_refiment_helper import *
+import color_refinement
+import tests
 from graph_io import *
-from tools import dot_to_pdf
 
-PATH = 'graphs/colorref'# to run locally from PyCharm: PATH = '../../graphs/colorref'
```

Comment:
Lokaal runnen kan nu met de `Run all tests in IDE` / `Run all tests in shell` run configs. Als je je eigen run config wilt draaien voor alleen dit bestand, dan moet de working dir van die config op de project root staan en niet een submap.